JOB.env

```diff
@@ -1,3 +1,3 @@
 DEFRA_VERSION=0.2.2
-BASE_VERSION=11.0.15-slim
+BASE_VERSION=11.0.16-slim
```

Comment:
Did you mean to leave the Defra version at 0.2.2?

Reply:
Apologies, no, fixed.